modules/gapi/src/api/python_bridge.cpp

```diff
@@ -0,0 +1,50 @@
+//This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include <opencv2/gapi/python_bridge.hpp>
+
+void cv::detail::allocateGraphOutputs(const cv::GTypesInfo &out_info,
+                                            cv::GRunArgs   &args,
+                                            cv::GRunArgsP  &outs)
+{
+    for (auto&& info : out_info)
+    {
+        switch (info.shape)
+        {
+            case cv::GShape::GMAT:
+            {
+                args.emplace_back(cv::Mat{});
```

Comment:
Oh this is not even an _allocation_. I'd say it just constructs the output vector with the right shape.

Reply:
Fixed name